Resfeber/View Controllers/FavoritesViewController.swift

```diff
@@ -91,8 +101,8 @@ extension FavoritesViewController: UICollectionViewDataSource {
 
 extension FavoritesViewController: UICollectionViewDelegate {
     func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
-        guard indexPath.row < DestinationData.favoriteDestinations.count else { return }
-        let destination = DestinationData.favoriteDestinations[indexPath.row]
+        guard indexPath.row < destinationController.favoriteDestinations.count else { return }
```

Comment:
If the collection view is a true reflection of the items in your data source, this should never be a problem right? If you were seeing this crash earlier, it probably means something is wrong with your data source

Reply:
Agreed. I put this extra check in when the data source was not properly synchronized with the view controller to communicate changes to the data. The sync issue has since been resolved and the extra guard statement was removed.